platform/app/public/config/default.js

```diff
@@ -50,7 +50,7 @@ window.config = {
         imageRendering: 'wadors',
         thumbnailRendering: 'wadors',
         enableStudyLazyLoad: true,
-        supportsFuzzyMatching: false,
+        supportsFuzzyMatching: true,
```

Comment:
Suggest changing the next line for supportsWildcard to false, or else implement the logic for wildcard matching too.

Reply:
done